generate.sh

```diff
@@ -109,7 +103,7 @@ mkdir -p _themes
 
 # Setting up documentation sources
 print_log "Setting up documentation sources\n"
-sphinx-quickstart -q -v "$VERSION" -a "$AUTHOR" -p "$PROJECTNAME" -t templates/ -d html_theme=$DOCTHEME -d html_logo=$LOGO -d root_dir=$ROOT_DIR -d autoapi_python=$AUTOAPI_PYTHON -d mock_modules=$MOCK_MODULES >> ${LOGFILE}
+sphinx-quickstart -q -v "$VERSION" -a "$AUTHOR" -p "$PROJECTNAME" -t templates/ -d html_theme=$DOCTHEME -d html_logo=$LOGO -d root_dir=$ROOT_DIR -d autoapi_python=$AUTOAPI_PYTHON -d mock_modules=$MOCK_MODULES >>${LOGFILE} 2>>${LOGFILE}
```

Comment:
I tried removing this, but then the script hanged at this location. I think maybe it asks for some user input when we remove `-q`